arcgis-ios-sdk-samples/Edit data/Add features with contingent values/AddFeaturesContingentValuesViewController.swift

```diff
@@ -0,0 +1,200 @@
+        // Create an array of graphics to add to the graphics overlay.
+        var graphics = [AGSGraphic]()
+        // Query the features with the query parameters.
+        featureTable?.queryFeatures(with: queryParameters) { [weak self ] result, error in
+            guard let self = self else { return }
+            if let error = error {
+                // Present an alert if there is an error while querying the features.
+                self.presentAlert(error: error)
+            } else if let result = result {
+                // Get the array of features from the query result.
+                let features = result.featureEnumerator().allObjects
+                // For each feature, add the buffer symbol according to its buffer size.
+                features.forEach { feature in
+                    let graphic = self.createGraphics(for: feature)
+                    graphics.append(graphic)
+                }
+                // Add the graphics to the graphics overlay.
+                self.graphicsOverlay.graphics.addObjects(from: graphics)
+                // Add the graphics overlay to the map view.
+                self.mapView.graphicsOverlays.add(self.graphicsOverlay)
+            }
+        }
+    }
+    
+    /// Add a single feature to the map.
+    func addFeature(at mapPoint: AGSPoint) {
+        // Create a symbol to represent a bird's nest.
+        let symbol = AGSSimpleMarkerSymbol(style: .circle, color: .black, size: 11)
+        // Add the graphic to the graphics overlay.
+        graphicsOverlay.graphics.add(AGSGraphic(geometry: mapPoint, symbol: symbol))
+        // Show the attributes table view.
+        performSegue(withIdentifier: "AddFeature", sender: self)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Set the map view's touch delegate.
+        mapView.touchDelegate = self
+        
+        // Add the source code button item to the right of navigation bar.
+        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = [
+            "AddFeaturesContingentValuesViewController",
+            "ContingentValuesTableViewController"
+        ]
+    }
+    
+    // MARK: - Navigation
+    
+    /// Prepare for the segue.
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Set the controller and its properties.
+        if let navigationController = segue.destination as? UINavigationController,
+           let controller = navigationController.viewControllers.first as? ContingentValuesTableViewController {
+            controller.isModalInPresentation = true
+            controller.featureTable = featureTable
+            controller.graphicsOverlay = graphicsOverlay
+            controller.mapPoint = mapPoint
+            controller.delegate = self
+        }
+    }
+}
+
+extension AddFeaturesContingentValuesViewController: ContingentValuesDelegate {
+    func createGraphics(for feature: AGSFeature) -> AGSGraphic {
```

Comment:
```suggestion
    func createGraphic(for feature: AGSFeature) -> AGSGraphic {
```
Don't know if this is a correct suggestion haha 😅